### ISSUE TITLE
Failed to run ti i18n extract -l en -a OR ti-i18n extract -l en -a

### ISSUE THREAD

**Issue by caspahouzer** [FokkeZB/ti-i18n#16]
With version 0.5.1 I got the following error:

~~~
/usr/local/lib/node_modules/ti-i18n/lib/strings.js:68
		var value = doc.createTextNode(normalize(val));
		                               ^
[ERROR] Failed to run command "i18n"
[ERROR] ReferenceError: normalize is not defined
    at /usr/local/lib/node_modules/ti-i18n/lib/strings.js:68:34
    at _.each._.forEach (/usr/local/lib/node_modules/ti-i18n/node_modules/underscore/underscore.js:145:9)
    at exports.append (/usr/local/lib/node_modules/ti-i18n/lib/strings.js:66:4)
    at /usr/local/lib/node_modules/ti-i18n/lib/extract.js:247:13
    at _.each._.forEach (/usr/local/lib/node_modules/ti-i18n/node_modules/underscore/underscore.js:145:9)
    at exports.run (/usr/local/lib/node_modules/ti-i18n/lib/extract.js:236:4)
    at exports.extract (/usr/local/lib/node_modules/ti-i18n/main.js:69:30)
    at exports.run (/usr/local/lib/node_modules/ti-i18n/commands/i18n.js:39:21)
    at CLI.<anonymous> (/usr/local/lib/node_modules/titanium/lib/cli.js:930:5)
    at /usr/local/lib/node_modules/titanium/node_modules/async/lib/async.js:119:25
---------------------------------------------
    at Object.run (/usr/local/lib/node_modules/titanium/node_modules/node-appc/lib/subprocess.js:59:8)
    at find (/usr/local/lib/node_modules/titanium/node_modules/node-appc/lib/subprocess.js:96:14)
    at q.process (/usr/local/lib/node_modules/titanium/node_modules/node-appc/node_modules/async/lib/async.js:809:21)
    at processImmediate (timers.js:330:15)
~~~

**Comment by FokkeZB** [comment]
Fixed in 0.5.2

Sorry... going too fast here and this module doesn't have proper test unfortunately.

**Comment by caspahouzer** [comment]
Works again. Thanks.
